static/app/views/alerts/rules/metric/wizardField.tsx

```diff
@@ -125,7 +125,11 @@ export default function WizardField({
             AlertWizardRuleTemplates,
             template =>
               template.aggregate === aggregate &&
-              template.dataset === dataset &&
+              (template.dataset === dataset ||
+                (organization.features.includes('alert-crash-free-metrics') &&
+                  (template.aggregate === SessionsAggregate.CRASH_FREE_SESSIONS ||
```

Comment:
This doesn't end up looking nice because `template.aggregate` is `string` whereas the other two are `SessionsAggregate`, have to do something hacky like `(([SessionsAggregate.CRASH_FREE_SESSIONS, SessionsAggregate.CRASH_FREE_USERS] as string[])`, I'd rather have it as is